src/main/java/com/team4/testingsystem/services/impl/TestEvaluationServiceImpl.java

```diff
@@ -80,4 +104,12 @@ public void updateScoreAfterCoachCheck(Test test) {
         saveScoreCoachCheck(test, Modules.ESSAY, gradeMap);
         saveScoreCoachCheck(test, Modules.SPEAKING, gradeMap);
     }
+
+    @Override
+    public void updateCoachAnswersAfterCheck(Test test) {
+        List<CoachAnswer> allCoachAnswers = coachAnswerRepository.findAllById_Test(test);
+
+        saveAnswersCoachCheck(Modules.GRAMMAR, allCoachAnswers);
+        saveAnswersCoachCheck(Modules.LISTENING, allCoachAnswers);
```

Comment:
What coach comments do you expect on grammar or listening modules?

Reply:
I'm a little confused) I'll return everything as it was and use the controller.